recipes/libhwloc/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "libhwloc" %}
+{% set pname = "hwloc" %}
+{% set version = "1.11.4" %}
+{% set sha256 = "1b6a58049c31ce36aff162cf4332998fd468486bd08fdfe0249a47437311512d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ pname }}-{{ version }}.tar.gz
+  url: https://www.open-mpi.org/software/hwloc/v1.11/downloads/{{ pname }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - gcc
```

Comment:
Are you using `Fortran` or `libgomp`? If not you do not need the `gcc` package. If you do then you need to add `libgcc` or `libgfortran` (former catches all cases the latter if only `Fortran` is needed) as a run dependency.